Library/Formula/texmath.rb

```diff
@@ -0,0 +1,30 @@
+require "language/haskell"
+
+class Texmath < Formula
+  include Language::Haskell::Cabal
+
+  homepage "http://johnmacfarlane.net/texmath.html"
+  url "https://hackage.haskell.org/package/texmath-0.8.0.2/texmath-0.8.0.2.tar.gz"
+  sha1 "9d7058b2c92ff1dd2f9a40ae65ecbc30de23abc8"
+
+  depends_on "ghc" => :build
+  depends_on "cabal-install" => :build
+  depends_on "gmp"
+
+  fails_with(:clang) { build 425 } # clang segfaults on Lion
+
+  def install
+    cabal_sandbox do
+      cabal_install "--only-dependencies"
+      cabal_install "--prefix=#{prefix}"
+      cabal_install "-fexecutable"
+    end
+    cabal_clean_lib
+  end
+
+  test do
+    system "echo 'a^2 + b^2 = c^2' > pythagorean.tex"
+    system "texmath pythagorean.tex > pythagorean.xml"
+    assert (Pathname.pwd/"pythagorean.xml").read.include? "<mn>2</mn>"
```

Comment:
`assert_match "<mn>2</mn>", shell_output("texmath pythagorean.tex")` rather than the above two lines.